v2/apiserver/internal/core/common_test.go

```diff
@@ -131,7 +131,7 @@ type mockSubstrate struct {
 		project Project,
 	) (Project, error)
 	DeleteProjectFn       func(context.Context, Project) error
-	CreateEventFn         func(context.Context, Project, Event) (Event, error)
```

Comment:
This got removed because the interface this mock fulfills no longer requires it-- in fact it hasn't for a while, so this is dead code.